.github/actions/package/expected-checksec.json

```diff
@@ -0,0 +1,10 @@
+{
+  "canary": "yes",
+  "fortify_source": "yes",
+  "nx": "yes",
+  "pie": "yes",
+  "relro": "full",
+  "rpath": "no",
+  "runpath": "no",
+  "symbols": "no"
```

Comment:
nit/tioli: it might be nice to have comments explaining why we expect to see "no" for these?

Reply:
I didn't do too much research into all of the various checks checksec does, but i'd welcome additional docs.

I think in the case of _symbols_, particularly, the guidance is intended to make reverse-engineering more difficult. In our case, we mostly care that the binary is small (and we have another build option that includes symbols as a separate file)

Reply:
It appears that rpath/runpath can be used to influence dynamic linking in dangerous ways...

Reply:
yeah, i thought that was the case for symbols. I wasn't super familiar with the rpath/rumpath stuff.